hezronkimutai/PropertyProLiteAPI
# #167165112 Deploy app to heroku

### What does this PR do? 
This PR deploys the latest version of the application to heroku and corrects the faulty routes
#### Description of Task to be completed?
- Added some isAdmin and address fields to user signup route.
- Corrected the dummy data in the tests.
- Deploy the app to heroku.
#### How should this be manually tested?
- Clone this repository.
- checkout to  ch-heroku-167165112 branch.
- Run the npm run testv1 to see the tests.
- Test the routes on heroku.
- You might as well use [This](https://propertyproliteapi.herokuapp.com/) heroku link. 
#### What are the relevant pivotal tracker stories?
[#167165112](https://www.pivotaltracker.com/n/projects/2353827/stories/167165112)
